### ISSUE TITLE
dbm.error: db type is dbm.gnu, but the module is not available

### ISSUE THREAD

**Issue by rhubert** [BobBuildTool/bob#183]
```
Traceback (most recent call last):
  File "/home/hubert/projects/bob/pym/bob/scripts.py", line 119, in catchErrors
    ret = fun(*args, **kwargs)
  File "/home/hubert/projects/bob/pym/bob/scripts.py", line 219, in cmd
    cmd(args.args, bobRoot)
  File "/home/hubert/projects/bob/pym/bob/scripts.py", line 57, in __ls
    doLS(*args, **kwargs)
  File "/home/hubert/projects/bob/pym/bob/cmds/misc.py", line 105, in doLS
    recipes.parse()
  File "/home/hubert/projects/bob/pym/bob/input.py", line 2267, in parse
    self.__cache.open()
  File "/home/hubert/projects/bob/pym/bob/input.py", line 2544, in open
    self.__shelve = shelve.open(".bob-cache.shelve")
  File "/usr/lib/python3.5/shelve.py", line 243, in open
    return DbfilenameShelf(filename, flag, protocol, writeback)
  File "/usr/lib/python3.5/shelve.py", line 227, in __init__
    Shelf.__init__(self, dbm.open(filename, flag), protocol, writeback)
  File "/usr/lib/python3.5/dbm/__init__.py", line 91, in open
    "available".format(result))
dbm.error: db type is dbm.gnu, but the module is not available
```

**Comment by jkloetzke** [comment]
Looks like Python was upgraded and one of the key-value-databases could not be read anymore. I really start to think about using alternatives to the `dbm` module. It feels really brittle. Bugs #171 and #118 are other examples. :thinking:

**Comment by rhubert** [comment]
One more reason to switch to a more stable db:

```
$ bob ls
Speicherzugriffsfehler
$ dmesg | tail
[520402.960494] python3[7112]: segfault at 0 ip 00007fa3ade1e73f sp 00007ffdfd8f5680 error 6 in _dbm.cpython-35m-x86_64-linux-gnu.so[7fa3ade1d000+3000]
[520435.756729] python3[7118]: segfault at 0 ip 00007f26492d873f sp 00007ffee6b60690 error 6 in _dbm.cpython-35m-x86_64-linux-gnu.so[7f26492d7000+3000]
[520461.737688] python3[7158]: segfault at 0 ip 00007ff2e2cbe73f sp 00007ffedb61c430 error 6 in _dbm.cpython-35m-x86_64-linux-gnu.so[7ff2e2cbd000+3000]
```

**Comment by jkloetzke** [comment]
Fixed in 5b49c6f0a8ff2c57acff9000631a9e43efcd6ed6.

**Comment by valleu326** [comment]
delete comfyui-workspace-manager node, and reinstall it again.
